core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -34,6 +34,11 @@ class SearchService(
     with FileChangeListener
     with SLF4JLogging {
 
+  private[indexer] def isUserFile(file: FileName): Boolean = {
+    //may want to cache the mapped results. 
+    (config.allTargets map { vfs.vfile }) exists { file isAncestor _.getName }
```

Comment:
when using the syntax sugar for function application, always use `(...)` instead of `{...}` because there are some corner cases about how the desugaring works.